### PR TITLE
Fix incorrect information about SCIM

### PR DESCRIPTION
**Pull request by datanerd**
### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by andrekolodochka** [comment]
I checked the results in staging and looks right.
There's only one thing I'd change, but I'm not sure whether it worth it. I've added "If SAML SSO is configured on organisation" languages as I can see that the same data/reusables/saml/about-linked-identities.md templated is used in both org and enterprise account related articles. Ideally we should have two separate templates, but I suspect it'll require a review of all documentation and updating it with either one or another.

**Comment by ramyaparimi** [comment]
@andrekolodochka 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/saml/about-linked-identities.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account. When available, the entry will include SCIM data. For more information, see "[About SCIM](/organizations/managing-saml-single-sign-on-for-your-organization/about-scim)."
+You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account. If SAML SSO is configured on organisation, when available, the entry will include SCIM data. For more information, see "[About SCIM](/organizations/managing-saml-single-sign-on-for-your-organization/about-scim)."
```
Comment (lecoursen) [inline data/reusables/saml/about-linked-identities.md:1]: Let's just remove the SCIM part from the reusable and add it directly to "Viewing and managing a member's SAML access to your organization" right after the reference to the reusable.

```suggestion
You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account.
```

File: content/organizations/granting-access-to-your-organization-with-saml-single-sign-on/viewing-and-managing-a-members-saml-access-to-your-organization.md [hunk 2]
```diff
@@ -25,6 +25,16 @@ You can view and revoke each member's linked identity, active sessions, and auth
 
 {% data reusables.saml.about-linked-identities %}
 
+{% warning %}
+
+**Warning:** For organizations using SCIM:
+- Revoking a linked user identity on {% data variables.product.product_name %} will also remove the SAML and SCIM metadata. As a result, the identity provider will not be able to synchronize or deprovision the linked user identity.
+- An admin must revoke a linked identity through the identity provider.
+- To revoke a linked identity and link a different account through the identity provider, an admin can remove and re-assign the user to the {% data variables.product.product_name %} application. For more information, see your identity provider's docs.
```
Comment (lecoursen) [inline content/organizations/granting-access-to-your-organization-with-saml-single-sign-on/viewing-and-managing-a-members-saml-access-to-your-organization.md:33]:
```suggestion
- To revoke a linked identity and link a different account through the identity provider, an admin can remove and re-assign the user to the {% data variables.product.product_name %} application. For more information, see your identity provider's documentation.
```

**Review by lecoursen — COMMENTED** [review]
Thanks @andrekolodochka! I'm going to apply a few tiny suggestions and then merge! ⚡

**Review by lecoursen — DISMISSED** [review]

**Review by lecoursen — APPROVED** [review]

**Pull request merged**